Makefile

```diff
@@ -111,28 +109,23 @@ build:
 
 .PHONY: start-node
 start-node:
-	@echo "Build ${BUILD_PATH}"
-	@echo "Build ${CONFIG_PATH}"
-	@echo "Build ${CONFIG_PATH2}"
-	@echo "Command ${NODE_COMMAND}"
+	@echo "Build binary: ${BUILD_PATH}"
+	@echo "Config path: ${CONFIG_PATH}"
+	@echo "Share config path: ${SHARE_CONFIG}"
+	@echo "Command provided: ${NODE_COMMAND_ARGS}"
 ifdef DEBUG_PORT
 	@echo "Running node-${NODE_ID} in debug mode"
 	@dlv  --continue --accept-multiclient --headless --listen=:${DEBUG_PORT} --api-version=2 exec \
-	 ${BUILD_PATH} start-node -- ${NODE_COMMAND}
+	 ${BUILD_PATH} start-node -- ${NODE_COMMAND_ARGS}
 else
-	@echo "Running node on address: ${HOST_ADDRESS})"
-	@${BUILD_PATH} start-node ${NODE_COMMAND}
+	@echo "Running node on address: ${HOST_ADDRESS}"
+	@${BUILD_PATH} start-node ${NODE_COMMAND_ARGS}
 endif
 
 .PHONY: docker
 docker:
 	@echo "node ${NODES_ID}"
-	@docker rm -f ssv_node && docker build -t ssv_node . && docker run -d --env-file .env --restart unless-stopped --name=ssv_node -p 13000:13000 -p 12000:12000/udp -it ssv_node make BUILD_PATH=/go/bin/ssvnode  start-node && docker logs ssv_node --follow
-
-.PHONY: docker-image
-docker-image:
-	@echo "node ${NODES_ID}"
-	@sudo docker rm -f ssv_node && docker run -d --env-file .env --restart unless-stopped --name=ssv_node -p 13000:13000 -p 12000:12000/udp 'ssvlabs/ssv-node:latest' make BUILD_PATH=/go/bin/ssvnode start-node
+	@docker rm -f ssv_node && docker build -t ssv_node . && docker run --env-file .env --name=ssv_node -p 16000:16000 -p 13001:13001 -p 12001:12001/udp -v ./data:/data -it ssv_node make BUILD_PATH=/go/bin/ssvnode start-node && docker logs ssv_node --follow
```

Comment:
I feel like these docker commands are outdated - so I'm replacing these with a new one that I found to be useful for testing locally.